pkgs/desktops/gnome/core/gnome-desktop/default.nix

```diff
@@ -75,6 +75,8 @@ stdenv.mkDerivation rec {
   mesonFlags = [
     "-Dgtk_doc=true"
     "-Ddesktop_docs=false"
+    "-Ddate_in_gnome_version=false"
```

Comment:
Nit: Is it possible to set the date somehow? I'd prefer if it embedded `SOURCE_DATE_EPOCH`.

Reply:
there is no option for that https://gitlab.gnome.org/GNOME/gnome-desktop/-/blob/master/meson_options.txt